aiokafka/coordinator/assignors/sticky/sticky_assignor.py

```diff
@@ -170,14 +200,14 @@ def balance(self):
             self.current_assignment[consumer] = partitions
             self._add_consumer_to_current_subscriptions_and_maintain_order(consumer)
 
-    def get_final_assignment(self, member_id):
-        assignment = defaultdict(list)
+    def get_final_assignment(self, member_id: str) -> Dict[str, List[int]]:
+        assignment: Dict[str, List[int]] = defaultdict(list)
         for topic_partition in self.current_assignment[member_id]:
             assignment[topic_partition.topic].append(topic_partition.partition)
         assignment = {k: sorted(v) for k, v in assignment.items()}
-        return assignment.items()
+        return assignment
```

Comment:
This is incompatible change. Could you justify, why this way it's better?

Reply:
Reverted